src/index.css

```diff
@@ -0,0 +1,9 @@
+@tailwind base;
```

Comment:
Please use the `main.sass`for global stylings

Reply:
I guess, this was when you changed the file configuration to sass as it was initially index.css when I setup the tailwindcss
I would remove that now